docs/src/pages/demos/selection-controls/RadioButtonsGroup.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { makeStyles, Theme } from '@material-ui/core/styles';
+import Radio from '@material-ui/core/Radio';
+import RadioGroup from '@material-ui/core/RadioGroup';
+import FormHelperText from '@material-ui/core/FormHelperText';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import FormControl from '@material-ui/core/FormControl';
+import FormLabel from '@material-ui/core/FormLabel';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  root: {
+    display: 'flex',
+  },
+  formControl: {
+    margin: theme.spacing(3),
+  },
+  group: {
+    margin: theme.spacing(1, 0),
+  },
+}));
+
+function RadioButtonsGroup() {
+  const classes = useStyles();
+  const [value, setValue] = React.useState('female');
+
+  function handleChange(event: React.ChangeEvent<any>) {
```

Comment:
Would it work? Do we need any?

```suggestion
  function handleChange(event: React.ChangeEvent<HTMLInputElement>) {
```

Reply:
No it does not work, I kept getting this error: 

```
10-17-178-40:selection-controls brandonherrera$ yarn docs:typescript:check
yarn run v1.15.2
$ tslint -p docs/tsconfig.json

ERROR: /Users/brandonherrera/material-ui/docs/src/pages/demos/selection-controls/RadioButtonsGroup.tsx:39:11 - TypeScript@next compile error: 
Type '(event: ChangeEvent<HTMLInputElement>) => void' is not assignable to type '(event: ChangeEvent<{}>, value: string) => void'.
  Types of parameters 'event' and 'event' are incompatible.
    Type 'ChangeEvent<{}>' is not assignable to type 'ChangeEvent<HTMLInputElement>'.
      Type '{}' is missing the following properties from type 'HTMLInputElement': accept, align, alt, autocomplete, and 289 more.
ERROR: /Users/brandonherrera/material-ui/docs/src/pages/demos/selection-controls/RadioButtonsGroup.tsx:59:11 - TypeScript@next compile error: 
Type '(event: ChangeEvent<HTMLInputElement>) => void' is not assignable to type '(event: ChangeEvent<{}>, value: string) => void'.

error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```
I used "any" because it was suggested in #13065.

Reply:
Use the `any` cast as late as possible. Rather `React.ChangeEvent<unknown>` and the cast to `any` when you need to pass `event.target.value` around.

Reply:
@eps1lon Just to clarify, are you suggesting to change from

```typescript
function handleChange(event: React.ChangeEvent<any>) {
     setValue(event.target.value);
}
```
to this?

```typescript
function handleChange(event: React.ChangeEvent<unknown>) {
     setValue((event as any).target.value);
}
```

Reply:
Almost. With "as late as possible" I meant `event.target.value as string`.

Reply:
> it's technically allowed to attach the handler to other element types.

Is that different from the Switch or Checkbox components? What do you mean by attaching the handler to another element type?